config/jobs/build-drivers/build-new-amazonlinux.yaml

```diff
@@ -148,13 +148,13 @@ postsubmits:
     agent: kubernetes
     branches:
       - ^master$
-    run_if_changed: '^driverkit/config/[a-z0-9.+-]{5,}/aarch64/amazonlinux_*'
+    run_if_changed: '^driverkit/config/[a-z0-9.+-]{5,}/aarch64/amazonlinux2_*'
```

Comment:
Fixed...we never published any amazonlinux2 artifact since we split arm64 jobs on native nodes :/